indico/modules/events/layout/default.py

```diff
@@ -178,7 +178,7 @@ def get_default_menu_entries():
         MenuEntryData(
             title=_("My Sessions"),
```

Comment:
This should go  to the sessions module (using `signals.event.sidemenu`). Only legacy menu entries are in `layout/default.py`.